example.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html ng-app="inavApp">
```

Comment:
On the page you sent me, the ng-app was defined on the body. 

Per the Angular 1.5.0 tutorials, often you'll see this at the HTML level. 
https://docs.angularjs.org/tutorial/step_02